### PR TITLE
chezmoi: add page

### PR DESCRIPTION
**Pull request by lukebrb**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/chezmoi.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+> A multi-machine dotfile manager, written in Go.
+> More information: <https://github.com/twpayne/chezmoi>.
+
+- Initialize chezmoi on your machine:
+
+`chezmoi init`
+
+- Tell chezmoi to manage a dotfile:
+
+`chezmoi add {{path_to_file}}`
```
Comment (navarroaxel) [inline pages/common/chezmoi.md:12]: we use `/`:
```suggestion
`chezmoi add {{path/to/file}}`
```

File: pages/common/chezmoi.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Edit the source state of a tracked dotfile:
+
+`chezmoi edit {{path_to_file}}`
```
Comment (navarroaxel) [inline pages/common/chezmoi.md:16]:
```suggestion
`chezmoi edit {{path/to/file}}`
```

File: pages/common/chezmoi.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`chezmoi diff`
+
+- Apply the changes:
+
+`chezmoi -v apply`
+
+- Set chezmoi up on another machine, from a service like GitHub:
+
+`chezmoi init {{https://git_url}}`
```
Comment (navarroaxel) [inline pages/common/chezmoi.md:28]:
```suggestion
`chezmoi init {{https://example.com/repository.git}}`
```

File: pages/common/chezmoi.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- See changes chezmoi would make:
+
+`chezmoi diff`
+
+- Apply the changes:
+
+`chezmoi -v apply`
+
+- Set chezmoi up on another machine, from a service like GitHub:
```
Comment (navarroaxel) [inline pages/common/chezmoi.md:26]: I'm not sure about this description, the official site says:
```
Clone the git repo in ~/.local/share/chezmoi to a hosted Git service, e.g. GitHub, GitLab, or BitBucket. Many people call their dotfiles repo dotfiles. You can then setup chezmoi on a second machine:
```

Is this command for download dotfiles from your git repository?

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Comment by navarroaxel** [comment]
Thanks for you PR @lukebrb! I left some comments and questions.

File: pages/common/chezmoi.md (hunk 4 above)
Comment (sbrl) [reply pages/common/chezmoi.md:26]: It looks like it, @navarroaxel. What about this?
```suggestion
- Set chezmoi up on another machine by downloading existing dotfiles from a git repo:
```

File: pages/common/chezmoi.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`chezmoi init {{https://git_url}}`
+
+- Fetch latest changes from a remote repo:
```
Comment (sbrl) [inline pages/common/chezmoi.md:30]: Perhaps this will read better?
```suggestion
- Fetch the latest changes from a remote repository:
```

**Review by schneiderl — APPROVED** [review]
LGTM after the other comments are addressed.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**